doc/man3/CMS_EnvelopedData_create.pod

```diff
@@ -0,0 +1,52 @@
+=pod
+
+=head1 NAME
+
+CMS_EnvelopedData_create - Create CMS envelope
+
+=head1 SYNOPSIS
+
+ #include <openssl/cms.h>
+
+ CMS_ContentInfo *CMS_EnvelopedData_create(const EVP_CIPHER *cipher);
+
+=head1 DESCRIPTION
+
+CMS_EnvelopedData_create() creates a B<CMS_ContentInfo> structure with
+a type B<NID_pkcs7_enveloped>. B<cipher> is the symmetric cipher to use.
+
+The algorithm passed in the B<cipher> parameter must support ASN1 encoding of
+its parameters.
+
+The recipients can be added later using CMS_add1_recipient_cert() or
+CMS_add0_recipient_key().
```

Comment:
For references to functions on other man pages enclose them in an `L` tag, with the section number (3) in parens, i.e. `L<CMS_add1_recipient_cert(3)>`

Reply:
Should this be done for all mentioned functions? I have been actually looking through the docs before and it seems that it's mostly without any link (see for example docs CMS_encrypt). I can of course add it but just wondering if I should add links everywhere...

Reply:
Any function mentioned on  this page (except mentions of CMS_EnvelopedData_create itself) should be done in this way. There may well be discrepancies on other pages which don't do this - but this is the standard way we try to do it.